clpy/backend/function.pyx

```diff
@@ -114,12 +114,16 @@ cdef void _launch(clpy.backend.opencl.types.cl_kernel kernel, global_work_size,
     cdef size_t gws[3]
     for i in range(global_dim):
         gws[i] = global_work_size[i]
+    for i in range(global_dim+1, 3):
+        gws[i] = 1
```

Comment:
Why is this needed?

The value of `global_work_size[i]` should be ignored if `global_dim` is less.

Reply:
I misunderstood the relationship between global_dim and local_dim, which are asserted to be equal.
Removed it.